src/api/app/models/package.rb

```diff
@@ -1358,6 +1359,16 @@ def delete_from_sphinx
     DeleteFromSphinxJob.perform_later(id, self.class)
   end
 
+  def backfill_bs_request_actions
+    # Source package
+    BsRequestAction.where(source_project_id: nil, source_package_id: nil, source_project: project.name, source_package: name)
+                   .update!(source_package_id: id, source_project_id: project.id)
```

Comment:
```suggestion
    BsRequestAction.where(source_project: project.name, source_package: name)
                   .update_attribute(source_package_id: id)
```

Reply:
We want to update those associations, not back-fill. And let's do only `Package` related things here, we have `Project` related things covered in the `Project` callback.

Reply:
Suggestion applied.